### ISSUE TITLE
Could not run command 

### ISSUE THREAD

**Issue by wsaeed77** [mikehaertl/phpwkhtmltopdf#212]
I wanted to generate 5000 pages on the go using phpwkhtmltopdf. I am able to create 1200 pages at a time. If I try to create more then 1200 page it gives me an error "Could not run command". When I added style sheet to the pdf pages this error is generated on 800 pages. Please help me resolve the issue.

**Comment by mikehaertl** [comment]
Creating such huge PDFs is really an edge case. It's probably a limitation in `proc_open()`. With that many pages, the command to execute becomes really huge.

Have you tried to set `useExec` in the `commandOptions`?

```
$this->_pdf = new \mikehaertl\wkhtmlto\Pdf(array(
    'commandOptions' => array(
        'useExec' => true
    )
));
```

**Comment by mikehaertl** [comment]
Oh, and are you on windows?

**Comment by wsaeed77** [comment]
I am on linux centos 6.

**Comment by wsaeed77** [comment]
I have tried  'useExec' => true, but no success.

**Comment by mikehaertl** [comment]
Then I think, there's not really much we can do about it, sorry. 

For debugging you can try to echo the command with `echo $pdf->getCommand()->getExecCommand()` and check the length of this string. Also play around with it and see if you can call `wkhtmltopdf` from command line with such a long command. E.g. but it in a bash script file and see if that works.

But note that you may have to modify the command. Our library creates some temporary files with your HTML content, as `wkhtmltopdf` requires files as input.

**Comment by wsaeed77** [comment]
OK no problem. I will play around with the library and will let you know if I found any solution.

**Comment by mikehaertl** [comment]
This issue should be fixed with 2.4.1 so I'm closing it for now. Feel free to add feedback if the issue is still there.
